vendor/vendor.json

```diff
@@ -238,10 +238,10 @@
 			"revisionTime": "2018-01-25T22:56:34Z"
 		},
 		{
-			"checksumSHA1": "Jm51R9I7AKovceb8ZWXaBkLQCOE=",
+			"checksumSHA1": "grS9oX+0LtDbt/znfITZ5/CPc98=",
 			"path": "github.com/keybase/client/go/protocol/keybase1",
-			"revision": "97192f698f4d04d79a4b96fa73ac526aa6c20af7",
-			"revisionTime": "2018-03-20T23:57:06Z"
+			"revision": "2119e7ce2550b1e542dc6498d6710ef1a4b36838",
```

Comment:
I guess you'll fix this commit hash up after commiting keybase/client#11186?  Fine with me if you want to commit this first, then the other one, then fix this up in a new commit.

Reply:
I'll merge this one now, hoping to make client CI pass, merge CI, then make another commit here to update the hash.

Reply:
OK done!